CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## next
 
+* Added `include_nil` option to `View` for refining the rendering of nil values. 
+  * This can be set when defining the view with `Blueprint.view`, and defaults to false. For example: `view :default, include_nil: true { ... }`.
 * Fixed `Blueprint.describe` to output the proper `:id`. That is: the 'id` from the Class name, rather than the internal Struct attribute.
+* Fixed `Blueprint.dump(nil)` raising a `NoMethodError`
+
```

Comment:
Another thing we changed:

Fixed `Blueprint.load` with nil value to have a chance to return default values and recursing in the loading of the object.